app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Paulscupcakes</title>
+    <%= csrf_meta_tags %>
+
+    <%= javascript_include_tag 'application' %>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.1/css/materialize.min.css">
```

Comment:
Let's utilize the asset pipeline to bring in all of the CSS and JS assets. More info on why this is a best practice, and how to do so, [here](http://guides.rubyonrails.org/asset_pipeline.html)